crates/uv/tests/common/mod.rs

```diff
@@ -193,7 +223,7 @@ impl TestContext {
             .arg(self.cache_dir.path())
             .env("VIRTUAL_ENV", self.venv.as_os_str())
             .env("UV_NO_WRAP", "1")
-            .env("UV_TEST_PYTHON_PATH", "/dev/null")
+            .env("UV_TEST_PYTHON_PATH", &self.python_path())
```

Comment:
Using multiple Python versions is now "first class" so we can support filtering them.